src/main/java/com/zakem/RecipeCatalog/controller/IngredientController.java

```diff
@@ -0,0 +1,41 @@
+package com.zakem.RecipeCatalog.controller;
+
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.zakem.RecipeCatalog.object.Ingredient;
+import com.zakem.RecipeCatalog.repository.IngredientRepository;
+
+@RestController
+@RequestMapping("/ingredient")
+public class IngredientController {
+	
+	private final Logger logger = LoggerFactory.getLogger(this.getClass());
+	
+	IngredientRepository ingredientRepository;
+	
+	public IngredientController(IngredientRepository ingredientRepository) {
+		this.ingredientRepository = ingredientRepository;
```

Comment:
This is an example of something you would want to use `@Autowired` with. Then you wouldn't need a constructor.